CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ See `PUBLISH.md` for instructions on how to publish a new version.
 - (patch) Dependency updates
 - (minor) Generate TypeScript definitions for package
 - (patch) Dependency updates
+- (minor) New design system colors
```

Comment:
This should be in the unreleased changes section, not added to a release that has already been published